DER/CreditDerivatives/CreditDefaultSwaps.rdf

```diff
@@ -283,6 +308,21 @@
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">The notifying party is the party that notifies the other party when a credit or other triggering event has occurred by means of a credit event notice. If more than one party is referenced as being the notifying party then either party may notify the other of such an occurrence.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-cr-cds;SettlementAuction">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-stl;SettlementEvent"/>
+		<rdfs:label xml:lang="en">settlement auction</rdfs:label>
+		<skos:definition xml:lang="en">independently administered synthetic auction process on a set of defined deliverable obligations that sets the reference final price that can be used to facilitate cash settlement of all covered transactions following a credit event</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom>ISO 10962:2019, Securities and related financial instruments - Classification of financial instruments (CFI) code</fibo-fnd-utl-av:adaptedFrom>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-cr-cds;SingleNameCreditDefaultSwap">
+		<rdfs:subClassOf rdf:resource="&fibo-der-cr-cds;CreditDefaultSwap"/>
+		<rdfs:label xml:lang="en">single name credit default swap</rdfs:label>
+		<skos:definition xml:lang="en">credit default swap whose underlying risk is a single reference obligation, or a single reference entity, such as a corporation or a sovereign borrower</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom>Draft paper on Credit Default Swaps from the Federal Reserve Board, available at https://www.federalreserve.gov/econres/feds/files/2022023pap.pdf</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:adaptedFrom>ISO 10962:2019, Securities and related financial instruments - Classification of financial instruments (CFI) code</fibo-fnd-utl-av:adaptedFrom>
+	</owl:Class>
+	
```

Comment:
how are the types of triggering event associated with the CDS contract? Each TriggeringEvent occurrence needs to be linked to an event type linked with the contract

Reply:
@rivettp comment added to DER-139.